pkg/controllers/clusterinventory/clusterprofile/controller_integration_test.go

```diff
@@ -0,0 +1,128 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+package clusterprofile
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	rbacv1 "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	clusterinventory "sigs.k8s.io/cluster-inventory-api/apis/v1alpha1"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+
+	clusterv1beta1 "go.goms.io/fleet/apis/cluster/v1beta1"
+	"go.goms.io/fleet/pkg/utils"
+	"go.goms.io/fleet/pkg/utils/condition"
+)
+
+const (
+	testTargetCluster = "test-cluster"
+	clusterProfileNS  = "default"
+
+	eventuallyTimeout    = time.Second * 5
+	consistentlyDuration = time.Second * 10
+	interval             = time.Millisecond * 250
+)
+
+// This container cannot be run in parallel with other ITs because it uses a shared fakePlacementController.
+var _ = Describe("Test ClusterProfile Controller", func() {
+	var mc *clusterv1beta1.MemberCluster
+	var clusterProfile clusterinventory.ClusterProfile
+	var testMCName string
+	BeforeEach(func() {
+		testMCName = testTargetCluster + utils.RandStr()
+		By("Creating a new MemberCluster")
+		mc = clusterResourceBindingForTest(testMCName)
```

Comment:
Hi Ryan! Is the `clusterResourceBindingForTest()` function name correct here? I assume that it should create a member cluster object; the logic is correct, is just that the name was confusing